udapi/block/read/conllu.py

```diff
@@ -1,22 +1,17 @@
-import logging
 import re
-import bz2
 
 from udapi.core.basereader import BaseReader
 from udapi.core.root import Root
 
+# Compile a set of regular expressions that will be searched over the lines.
+RE_SENT_ID = re.compile(r'^# sent_id\s*=\s+(\S+)')
```

Comment:
Why did you add the `=` char to this pattern? All treebanks used by my project (CzEng, UDv1.4, ...) use
`# sent_id <sent_id>`